pkg/client/scanner/scanner.go

```diff
@@ -31,32 +35,30 @@ type Config struct {
 }
 
 // Available list all available scanner tools that can be used.
-var Available = [...]Tool{
-	Licensee,
-	Scancode,
-	Composer,
-	FileHasher,
-}
+var Available []Tool
 
 // Default is the default scanner tools that shall be used if no particular tool is selected.
-var Default Tool = Licensee
+var Default Tool
 
-// Licensee represents the latest usable Licensee version
-var Licensee = Tool{
-	Name:      "Licensee",
-	Version:   "9.13.0",
-	DockerImg: "docker.pkg.github.com/osrgroup/product-model-toolkit/scanner-licensee:9.13.0",
-	Cmd:       `/bin/bash -c "licensee detect /input/ --json > /result/result.json"`,
-	Results:   []string{"result.json"},
-}
+// init identifies available scanner tools
+func init() {
```

Comment:
How you want to manage the loaded scanner tools? Inside `scanner.go` are only pure functions who doesn't store any state.